controllers/networkchaos/trafficcontrol/types.go

```diff
@@ -128,9 +128,15 @@ func (r *endpoint) Apply(ctx context.Context, req ctrl.Request, chaos v1alpha1.I
 			return err
 		}
 	case v1alpha1.Both:
-		err = r.applyTc(ctx, pods, pods, externalCidrs, m, networkchaos)
+		err = r.applyTc(ctx, sources, targets, externalCidrs, m, networkchaos)
```

Comment:
There is another problem :crying_cat_face: . #1448 assumes that the ipset with the same name should have the same content. However, if you `applyTc` twice, they will have the same `ipset` name with different content.

```
dstIpset := ipset.BuildIPSet(targets, externalTargets, networkchaos, string(tcType)[0:5], m.Source)
```

The `postFix` should be modified from `string(tcType)[0:5]` to the `tgt`, `src` like the partition chaos

Reply:
Fixed

Reply:
Why `toTargetIPSetPostFix` and `bothTargetIPSetPostFix` are different? Should we use the same pattern for these names?